examples/exhibit-webpack/webpack.config.js

```diff
@@ -35,6 +35,5 @@ module.exports = {
   ]
 };
 
-// DELETE THIS LINE WHEN COPYING THIS EXAMPLE FOLDER OUTSIDE OF DECK.GL
-// It enables bundling against src in this repo rather than the installed version
-module.exports = require('../webpack.config.local')(module.exports);
+module.exports = process.env.NODE_ENV === 'development' ?
```

Comment:
I think I see why you made this change, and I think it is a good call, however:
 
1. This (support for local development using an optionally injected config) is a common pattern across all our examples in all our repos. Good to give it some extra thought. If this is good change maybe we want to do everywhere, e.g. before the coordinated release? 
3. In other repos, we use the webpack `--env` parameter (e.g. `--env.local`) instead of environment variables. Should we decide which way is better and use it everywhere?
4. I can see that the comment needed changing but I don't think we should delete it completely. There is a note at the top of the file, but it may not easily be spotted from here. We could at least give the user a hint what this strange construct is doing and that it is OK for the user to delete it. My view is that nothing should be considered trivial in webpack config files, every single line should have a comment :)